test.js

```diff
@@ -38,16 +38,22 @@ test('should return the right action when getting an action by key', t => {
   setConfig({
     actions: {
       'create-exchange': createExchangeRoles,
-      'delete-exchange': {
-        role: 'admin',
-        validate: (user, exchange) => exchange.user.id === user.id,
-      },
     },
   });
 
   t.equal(getActionByKey('create-exchange'), createExchangeRoles);
   t.end();
-})
+});
+
+test('should return false when action doesn\'t exist', t => {
+  setConfig({
+    actions: {},
+    role: 'role',
```

Comment:
set role to `admin` to make sure you test what you described.
this way it's also possible to have a side-effect because admin does not have the role named `role`